docker/etc/mime.types

```diff
@@ -0,0 +1,836 @@
+###############################################################################
+#
+#  MIME media types and the extensions that represent them.
+#
+#  The format of this file is a media type on the left and zero or more
+#  filename extensions on the right.  Programs using this file will map
+#  files ending with those extensions to the associated type.
+#
+#  This file is part of the "mime-support" package.  Please report a bug using
+#  the "reportbug" command of the "reportbug" package if you would like new
+#  types or extensions to be added.
```

Comment:
where is this file from?  From the comments it looks like it's been copied&pasted from another project?

Reply:
it comes from our docker image (default file in debian)

Reply:
I added to our repo so that we can reuse them in travis-ci

Reply:
_that's_ the part of the explanation I was missing 😉 

It's a pity we can't pull it in at travis build time so we don't have to keep it up to date

Reply:
Yep.. On the other hand, having it in our repo means a docker update won't break our detection (if the file gets updated in the debian image)